src/passes/MinifyImportsAndExports.cpp

```diff
@@ -179,13 +179,45 @@ struct MinifyImportsAndExports : public Pass {
     for (auto& pair : newToOld) {
       std::cout << pair.second.str << " => " << pair.first.str << '\n';
     }
+
+    if (minifyModules) {
+      doMinifyModules(module);
+    }
+  }
+
+  const Name SINGLETON_MODULE_NAME = "a";
```

Comment:
Nit: It should be spec-compliant to have this be the empty string, and that should save O(N) bytes :)

Though I'm not sure how JS-compliant it would be

Reply:
Interesting, yeah, I hadn't considered the empty string. It seems like it might work but I do worry about corner cases in VMs. I'll add a TODO.